src/plugin.js

```diff
@@ -121,6 +121,14 @@ const monitorPlayback = function(player, options) {
         resetMonitor();
       }
     });
+    healthcheck('adtimeupdate', function() {
```

Comment:
I talked this change over with Jon, and he thought this was a fine change. Since prepending `ad` onto all events is now a feature, I believe it is necessary to update errors to have an understanding of what ads are.